dependency_graph.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
```

Comment:
Is this graph produced on a regular basis? Or just on demand?
Would be nice to have this graphics included in the README or our documentation right?

Reply:
Just on demand as my plan was to put it in the wiki, right next to the PR testing instructions. But maybe the docs are a better place for both of them as it is documentation of the code. Good point! I'm going to migrate that soon.